frontstage/common/session.py

```diff
@@ -8,7 +8,6 @@ class Session(object):
     def __init__(self, session_key, encoded_jwt_token):
         self.encoded_jwt_token = encoded_jwt_token
         self.session_key = session_key
-        self.persisted = False
```

Comment:
This doesn't seem to have any use. It's a generic object so isn't a attribute of redis so seems historically, maybe it was useful at some stage

Reply:
From what I can see, this attribute was introduced solely in some slightly clunky logic around updating the Redis session with the message count (probably a workaround to a bug). Happy that's no longer required, and can be removed.

Reply:
My thoughts as well